luacompiler/src/lib/irgen/register_map.rs

```diff
@@ -76,6 +76,10 @@ impl<'a> RegisterMap<'a> {
     pub fn reg_count(self) -> usize {
         self.lifetimes.len()
     }
+
+    pub fn get_lifetimes(self) -> Vec<Lifetime> {
```

Comment:
Having a public function which consumes the `LuaToIR` feels weird: isn't the only caller of this going to be  `compile`?

Reply:
If it isn't public I cannot access it from `compile`.

Reply:
At least do `pub(crate)` then.